src/Controller/BarcodeController.php

```diff
@@ -76,25 +79,45 @@ public function barcodeAction(
      * Possible actions: ask_to_assign, ask_to_unassign
      * ask_to_assign: Will prompt the user to self-assign
```

Comment:
shall we have `ask_to_start` in the workflow? or maybe assign the task + start the task?

Reply:
Yeah good idea ! But i'm expecting changes here. I'm pretty sure the auto-assign on scan can fire back on multi points delivery